waspc/src/Wasp/Lib.hs

```diff
@@ -101,3 +114,25 @@ findMigrationsDir waspDir = do
   let migrationsAbsPath = waspDir SP.</> dbMigrationsDirInWaspProjectDir
   migrationsExists <- doesDirectoryExist $ SP.fromAbsDir migrationsAbsPath
   return $ if migrationsExists then Just migrationsAbsPath else Nothing
+
+loadDockerfileContents :: Path' Abs (Dir WaspProjectDir) -> IO (Maybe Text)
+loadDockerfileContents waspDir = do
+  let dockerfileAbsPath = SP.toFilePath $ waspDir SP.</> [relfile|Dockerfile|]
+  dockerfileExists <- doesFileExist dockerfileAbsPath
+  if dockerfileExists
+    then Just <$> T.IO.readFile dockerfileAbsPath
+    else return Nothing
```

Comment:
Oh wow, those are fancy! We actually have LambdaCase enabled by default now, so that is nice. Ok, I think I may pull in the `extra` package for the final form, that is nice and we probably will need other things from there in the future! 🚀

Reply:
Sounds good! Let's import `extra`, or we can even write our own function -> well probably easiest just to import it.